tutorials/openmm-torch-nnpops.ipynb

```diff
@@ -0,0 +1,440 @@
+      "execution_count": 3,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "D42I_L9r-Krj",
+        "outputId": "c1738d48-71e2-40ea-9965-d9dbd2147406"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "-1301523.8702643516\n"
+          ]
+        }
+      ],
+      "source": [
+        "# Comute the potential energy\n",
+        "pos = pt.tensor(ala2.positions.tolist())\n",
+        "energy_1 = float(nnp(pos))\n",
+        "print(energy_1)\n",
+        "\n",
+        "# Check if the energy is correct\n",
+        "assert pt.isclose(pt.tensor(energy_1), pt.tensor(-1301523.8703817206))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "zOV53fN2sB3C"
+      },
+      "source": [
+        "## Add the NNP to the system\n",
+        "\n",
+        "In order to use the NNP in a simulation, it has to loaded with [OpenMM-Torch](https://github.com/openmm/openmm-torch) and added to the system."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 4,
+      "metadata": {
+        "id": "Snj9UrPbsIS5"
+      },
+      "outputs": [],
+      "source": [
+        "from openmmtorch import TorchForce\n",
+        "\n",
+        "# Save the NNP to a file and load it with OpenMM-Torch\n",
+        "pt.jit.script(nnp).save('model.pt')\n",
+        "force = TorchForce('model.pt')\n",
+        "\n",
+        "# Add the NNP to the system\n",
+        "ala2.system.addForce(force)\n",
+        "assert ala2.system.getNumForces() == 1"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "jLtSbKg6BB1n"
+      },
+      "source": [
+        "## Setup a simulation\n",
+        "\n",
+        "Setup a simulation with [OpenMM](https://openmm.org/)."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 5,
+      "metadata": {
+        "id": "6l2vPxPZ3u56"
+      },
+      "outputs": [],
+      "source": [
+        "import sys\n",
+        "from openmm import LangevinMiddleIntegrator\n",
+        "from openmm.app import Simulation, StateDataReporter\n",
+        "from openmm.unit import kelvin, picosecond, femtosecond\n",
+        "\n",
+        "# Create an integrator with a time step of 1 fs\n",
+        "temperature = 298.15 * kelvin\n",
+        "frictionCoeff = 1 / picosecond\n",
+        "timeStep = 1 * femtosecond\n",
+        "integrator = LangevinMiddleIntegrator(temperature, frictionCoeff, timeStep)\n",
+        "\n",
+        "# Create a simulation and set the initial positions and velocities\n",
+        "simulation = Simulation(ala2.topology, ala2.system, integrator)\n",
+        "simulation.context.setPositions(ala2.positions)\n",
+        "# simulation.context.setVelocitiesToTemperature(temperature) # This does not work (https://github.com/openmm/openmm-torch/issues/61)\n",
```

Comment:
Does this work now that https://github.com/openmm/openmm-torch/issues/61 is closed?